x-pack/legacy/plugins/reporting/server/lib/create_queue.ts

```diff
@@ -5,13 +5,16 @@
  */
 
 import { PLUGIN_ID } from '../../common/constants';
+import { KbnServer } from '../../types';
+// @ts-ignore
 import { Esqueue } from './esqueue';
 import { createWorkerFactory } from './create_worker';
 import { oncePerServer } from './once_per_server';
 import { LevelLogger } from './level_logger';
+// @ts-ignore
 import { createTaggedLogger } from './create_tagged_logger'; // TODO remove createTaggedLogger once esqueue is removed
 
-function createQueueFn(server) {
+function createQueueFn(server: KbnServer): Esqueue {
```

Comment:
I think this is essentially the `any` type, because of the `ts-ignore` above the esqueue import.